sdk/core/azure-core/azure/core/exceptions.py

```diff
@@ -198,6 +198,7 @@ class AzureError(Exception):
     :ivar exc_traceback: The exc_traceback from sys.exc_info()
     :ivar exc_msg: A string formatting of message parameter, exc_type and exc_value
     :ivar str message: A stringified version of the message parameter
+    :ivar str continuation_token: A token reference to continue an incomplete operation
```

Comment:
Shall we write it's optional and not necessarly set?

Reply:
Yes.